Shared/Tracking/TrackerManager.swift

```diff
@@ -17,13 +17,15 @@ class TrackerManager {
     let anilist = AniListTracker()
     /// An instance of the MyAnimeList tracker.
     let myanimelist = MyAnimeListTracker()
+    /// An instance of the Komga tracker.
+    let komga = KomgaTracker()
 
     /// An array of the available trackers.
-    lazy var trackers: [Tracker] = [anilist, myanimelist]
+    lazy var trackers: [Tracker] = [anilist, myanimelist, komga]
 
     /// A boolean indicating if there is a tracker that is currently logged in.
-    var hasAvailableTrackers: Bool {
-        trackers.contains { $0.isLoggedIn }
+    func hasAvailableTrackers (manga: Manga?) -> Bool {
```

Comment:
default to providing no manga, since that was the prior behavior, and so you don't have to specify nil every time
```suggestion
    func hasAvailableTrackers(manga: Manga? = nil) -> Bool {
```